docs/details.hbs

```diff
@@ -0,0 +1,22 @@
+
+### {{ name }} ( [code](https://github.com/nymag/nymag-handlebars/blob/master/helpers/{{ category }}/{{ name }}.js) | {{#if hasTestFile}}[tests](https://github.com/nymag/nymag-handlebars/blob/master/helpers/{{ category }}/{{ name }}.test.js){{else}}no tests{{/if}} )
```

Comment:
can we make `no tests` big and red and flashing with a marching ants border? just an idea